group_vars/maas_pacemaker

```diff
@@ -8,4 +8,4 @@ maas_paf_revision: "1"
 maas_paf_deb_url: "https://github.com/ClusterLabs/PAF/releases/download/v{{ maas_paf_version }}/resource-agents-paf_{{ maas_paf_version }}-{{ maas_paf_revision }}_all.deb"
 maas_paf_deb_dest: "/tmp/resource-agents-paf_{{maas_paf_version }}-{{ maas_paf_revision }}_all.deb"
 
-maas_pacemaker_self_address: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] if 'ansible_default_ipv4' in hostvars[inventory_hostname] else hostvars['invenory_hostname']['ansible_default_ipv6']['address'] }}"
+maas_pacemaker_self_address: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] if 'ansible_default_ipv4' in hostvars[inventory_hostname] else hostvars['inventory_hostname']['ansible_default_ipv6']['address'] }}"
```

Comment:
hmm, `inventory_hostname` is not quoted in the ipv4 variant? which is it, quoted or not?

Reply:
not quoted, thanks for pointing it out